scripts/start-resource-server.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable consistent-return, no-console */
+
+const fs = require('fs');
+const { exec } = require('child_process');
+const path = require('path');
+
+// Users can also provide the testenv configuration at the root folder
+require('dotenv').config({ path: path.join(__dirname, '..', 'testenv') });
```

Comment:
Can you include `~/.okta/testenv` too?

similar to: https://github.com/okta/okta-oidc-tck/blob/master/e2e-tests/tools/common-config/index.js#L19-L21